packages/runtime-core/src/apiDefineComponent.ts

```diff
@@ -6,24 +6,24 @@ import {
   ComponentOptionsWithObjectProps,
   ComponentOptionsMixin,
   RenderFunction,
-  ComponentOptionsBase
+  ComponentOptionsBase,
```

Comment:
The prettier bump introduced unnecessary trailing comma format changes like this one. Please update `.prettierrc` with `trailingComma: 'none'` and remove these changes.